### PR TITLE
PATIOS 479: import localizations

### PR DESCRIPTION
**Pull request by diederikh**

### CODE REVIEW

File: ResearchKit/Localized/bn-IN.lproj/ResearchKit.strings [hunk 1]
```diff
@@ -0,0 +1,1479 @@
...
+"SPEECH_RECOGNITION_STOP_RECORD_LABEL" = "রেকর্ডিং বন্ধ করুন";
+
+/* (No Comment) */
+"SPEECH_RECOGNITION_TRANSCRIPTION_LABEL" = "আপনার প্রতিলিপি এখানে প্রদর্শিত হবে।";
+
+/* Speech Recognition active task. */
+"SPEECH_TASK_TITLE" = "স্পিচ রেকগনিশন";
+
+/* Progress indication (in navigation bar). */
+"STEP_PROGRESS_FORMAT" = "%2$@ এর % 1$@";
```
Comment (jcmontiel) [inline ResearchKit/Localized/bn-IN.lproj/ResearchKit.strings]: There's a 'Ulatus space' after the %

File: ResearchKit/Localized/de-AT.lproj/ResearchKit.strings [hunk 2]
```diff
@@ -0,0 +1,1479 @@
...
+"EMAIL_FORM_ITEM_TITLE" = "E-Mail Adresse";
+
+/* (No Comment) */
+"ENVIRONMENTSPL_CALCULATING" = "Berechnen";
+
+/* (No Comment) */
+"ENVIRONMENTSPL_INTRO_TEXT" = "Diese Aktivität misst den Umgebungsschalldruckpegel. Wenn die Geräuschpegel zu hoch sind, können Sie nicht fortfahren. Bitte gehen Sie an einen ruhigeren Ort, um diese Aufgabe zu erledigen oder wiederholen Sie diese Aktivität zu einem späteren Zeitpunkt.";
+
+/* (No Comment) */
+"ENVIRONMENTSPL_THRESHOLD" = "Schwelle: % @ dBA";
```
Comment (jcmontiel) [inline ResearchKit/Localized/de-AT.lproj/ResearchKit.strings]: 'Ulatus space'

File: ResearchKit/Localized/de-AT.lproj/ResearchKit.strings [hunk 3]
```diff
@@ -0,0 +1,1479 @@
...
+"FAMILY_NAME_ITEM_PLACEHOLDER" = "Appleseed";
+
+/* (No Comment) */
+"FITNESS_DISTANCE_TITLE_FORMAT" = "Entfernung (%@)";
+
+/* (No Comment) */
+"FITNESS_HEARTRATE_TITLE" = "Herzfrequenz (bpm)";
+
+/* (No Comment) */
+"FITNESS_INTRO_2_TEXT_FORMAT" = "Gehen Sie für %1$ @ im Freien in Ihrem schnellstmöglichen Tempo. Wenn Sie fertig sind, setzen Sie sich und ruhen Sie sich bequem für % 2$ @ aus. Um zu beginnen, tippen Sie auf Erste Schritte.";
```
Comment (jcmontiel) [inline ResearchKit/Localized/de-AT.lproj/ResearchKit.strings]: 'Ulatus space'

File: ResearchKit/Localized/de-AT.lproj/ResearchKit.strings [hunk 4]
```diff
@@ -0,0 +1,1479 @@
...
+
+/* (No Comment) */
+"FITNESS_INTRO_TEXT_FORMAT" = "Diese Aktivität überwacht Ihre Herzfrequenz und misst, wie weit Sie in % 1$ @ gehen können.";
```
Comment (jcmontiel) [inline ResearchKit/Localized/de-AT.lproj/ResearchKit.strings]: 'Ulatus space'

File: ResearchKit/Localized/de-AT.lproj/ResearchKit.strings [hunk 5]
```diff
@@ -0,0 +1,1479 @@
...
+"SHOULDER_RANGE_OF_MOTION_TOUCH_ANYWHERE_STEP_INSTRUCTION_LEFT" = "Legen Sie Ihr Gerät auf die linke Schulter";
+
+/* (No Comment) */
+"SHOULDER_RANGE_OF_MOTION_TOUCH_ANYWHERE_STEP_INSTRUCTION_RIGHT" = "Legen Sie Ihr Gerät auf die rechte Schulter";
+
+/* (No Comment) */
+"SPATIAL_SPAN_MEMORY_CALL_TO_ACTION" = "Tippen Sie zunächst auf Erste Schritte und schauen Sie genau hin.";
+
+/* (No Comment) */
+"SPATIAL_SPAN_MEMORY_INTRO_2_TEXT_%@" = "Einige der % @ leuchten nacheinander auf. Tippen Sie auf die % @ in der gleichen Reihenfolge, in der sie aufleuchteten.";
```
Comment (jcmontiel) [inline ResearchKit/Localized/de-AT.lproj/ResearchKit.strings]: 'Ulatus space' - multiple

File: ResearchKit/Localized/de-AT.lproj/ResearchKit.strings [hunk 6]
```diff
@@ -0,0 +1,1479 @@
...
+
+/* (No Comment) */
+"SPATIAL_SPAN_MEMORY_INTRO_2_TEXT_REVERSE_%@" = "Einige der % @ leuchten nacheinander auf. Tippen Sie auf %@ in der umgekehrten Reihenfolge, in der sie aufleuchteten.";
```
Comment (jcmontiel) [inline ResearchKit/Localized/de-AT.lproj/ResearchKit.strings]: 'Ulatus space'

File: ResearchKit/Localized/de-AT.lproj/ResearchKit.strings [hunk 7]
```diff
@@ -0,0 +1,1479 @@
...
+"TREMOR_TEST_ACTIVE_STEP_TOUCH_NOSE_INTRO" = "Halten Sie nun Ihren Ellbogen gebeugt und halten Sie Ihr Handy wiederholt an der Nase.";
+
+/* (No Comment) */
+"TREMOR_TEST_ACTIVE_STEP_TOUCH_NOSE_INTRO_LEFT" = "Halten Sie nun Ihren Ellbogen gebeugt, während Sie Ihr Handy in der LINKEN Hand halten. Halten Sie Ihr Handy wiederholt an der Nase.";
+
+/* (No Comment) */
+"TREMOR_TEST_ACTIVE_STEP_TOUCH_NOSE_INTRO_RIGHT" = "Halten Sie nun Ihren Ellbogen gebeugt, während Sie Ihr Handy in der RECHTEN Hand halten. Halten Sie Ihr Telefon wiederholt an der Nase.";
+
+/* (No Comment) */
+"TREMOR_TEST_ACTIVE_STEP_TURN_WRIST_INSTRUCTION_%ld" = "Führen Sie eine Queen-Wave für % ld Sekunden durch.";
```
Comment (jcmontiel) [inline ResearchKit/Localized/de-AT.lproj/ResearchKit.strings]: 'Ulatus space'

**Comment by diederikh** [comment]
@jcmontiel Good catch! Fixed.

**Review by dejan-medable — APPROVED** [review]

**Pull request merged**